cloudfront/wellcomelibrary.org/edge-lambda/src/api.test.ts

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+import {getWork} from "./api";
+import {testDataNoResults, testDataSingleResult} from './apiFixtures'
```

Comment:
This is a nit but you may want to get `prettier` autoformatting set up in here (will sort out spacing, single/double quotes, end-of-file newlines, etc etc).